server_installation/topics/operator/keycloak-backup-cr.adoc

```diff
@@ -0,0 +1,198 @@
+aws_s3_backup    0/1             6s           6s
+----
+
+. View the list of executed backup jobs. For example:
++
+[source,bash,subs=+attributes]
+----
+$ oc get pods
+NAME                               READY    STATUS       RESTARTS    AGE
+aws_s3_backup-5b4rfdd              0/1      Completed    0           24s
+keycloak-0                         1/1      Running      0           52m
+keycloak-postgresql-c824c6-vv27m   1/1      Running      0           71m
+----
+
+. View the log of your completed backup job:
++
+[source,bash,subs=+attributes]
+----
+$ oc logs aws_s3_backup-5b4rf
+==> Component data dump completed
+.
+.
+.
+.
+[source,bash,subs=+attributes]
+----
+
+The status of the backup job also appears in the AWS console.
+
+[[_backups-local-cr]]
+==== Backing up to Local Storage
+
+endif::[]
+You can the use Operator to create a backup job that performs a one-time backup to a local Persistent Volume. 
+
+.Example YAML file for a Backup custom resource
+```yaml
+apiVersion: keycloak.org/v1alpha1
+kind: KeycloakBackup
+metadata:
+  name: test-backup
+```
+
+.Prerequisites
+
+* You have a YAML file for this custom resource.
+ifeval::[{project_community}==true]
+Be sure to omit the `aws` sub-properties from this file.
+endif::[]
+
+* You have a `PersistentVolume` with a `claimRef` to reserve it only for a `PersistentVolumeClaim` created by the {project_name} Operator.
+
+.Procedure
+
+. Create a backup job: `oc create -f <backup_crname>. For example:
++
+[source,bash,subs=+attributes]
+----
+$ oc create -f one-time-backup.yaml
+westover.org/test-backup
+----
++ 
+The Operator creates a `PersistentVolumeClaim` with the following naming scheme:  `Keycloak-backup-<CR_name>`.
+
+. View a list of volumes:
++
+[source,bash,subs=+attributes]
+----
+$ oc get pvc
+NAME                          STATUS   VOLUME                 
+keycloak-backup-test-backup   Bound    pvc-e242-ew022d5-093q-3134n-41-adff
+keycloak-postresql-claim      Bound    pvc-e242-vs29202-9bcd7-093q-31-zadj
+----
+
+. View a list of backup jobs:
++
+[source,bash,subs=+attributes]
+----
+$ oc get jobs
+NAME           COMPLETIONS     DURATION     AGE
+test-backup    0/1             6s           6s
+----
+
+. View the list of executed backup jobs. For example:
++
+[source,bash,subs=+attributes]
+----
+$ oc get pods
+NAME                               READY    STATUS       RESTARTS    AGE
+test-backup-5b4rf                  0/1      Completed    0           24s
+keycloak-0                         1/1      Running      0           52m
+keycloak-postgresql-c824c6-vv27m   1/1      Running      0           71m
+----
+
+. View the log of your completed backup job:
++
+[source,bash,subs=+attributes]
+----
+$ oc logs test-backup-5b4rf
+==> Component data dump completed
+.
+.
+.
+.
+----
+
+.Additional resources
+
+For more details on persistent volumes, see the https://docs.okd.io/3.6/dev_guide/persistent_volumes.html#persistent-volumes-volumes-and-claim-prebinding[OKD manual].
```

Comment:
I do not think it is a good idea to have here such an old document. What do you think about this: https://kubernetes.io/docs/concepts/storage/persistent-volumes/?

Reply:
Okay, I can use that for the community, but what about for SSO.

Reply:
What about this one: https://docs.openshift.com/container-platform/4.4/storage/understanding-persistent-storage.html?